bot/controllers/channels.js

```diff
@@ -0,0 +1,85 @@
+module.exports = () => {
+  const util = require('apex-util');
+
+  const _run = (message) => {
+    const ctrls = [
+      {
+        cmd: '!chnl',
```

Comment:
I suggest using a full command name like !channel or something like !announce or !broadcast, more for consistency with other commands within this bot, and to convey the message as what the command does in one word.

Reply:
ok I made this change.